src/components/search-form/search-form.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import './search-form.css'
+
+const SearchForm = ({ initialValue, onSearch }) => {
+  const [searchTerm, setSearchTerm] = useState(initialValue || '');
+  const placeholder = 'What do you want to watch?';
+
+  const handleInputChange = () => {
```

Comment:
Naming issue, usually handleInputChange would refer to `onChange` listener for input, but here it's a submit handler

Reply:
Renamed to handleFormSubmit